bitswap/README.md

```diff
@@ -1,70 +1,78 @@
 
-
-# Status of this spec
+Status of this spec
+===================
 
 ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
 
-# Organization of this document
+Organization of this document
+=============================
 
 This spec is organized by chapters described on the *Table of contents* section. Each of the chapters can be found in its own file.
 
-# Table of contents
+Table of contents
+=================
 
-- [1]()
-- [2]()
-- [...]()
+-   [1]()
+-   [2]()
+-   [...]()
 
-# Introduction
+Introduction
+============
 
 Bitswap is IPFS' main block exchange protocol. It handles the requests made by a IPFS user, human or an application, to fetch data blocks from the network. It interacts with other Bitswap agents present in other IPFS nodes, exchanging (fetching + serving) blocks as it needs.
 
 Bitswap is a message based protocol, as opposed to response-reply. All messages contain wantlists, or blocks. Upon receiving a wantlist, an IPFS node should consider sending out wanted blocks if it has them. Upon receiving blocks, the node should send out a notification called a 'Cancel' signifying that they no longer want the block. At a protocol level, bitswap is very simple.
 
 Bitswap is a simple protocol overall. However, to make it fast and low on memory footprint, there are several implementation details that are important to get right. We document these details to our best capability on this document, so that other implementers can learn from the knowledge gather over the several iterations of bitswap.
 
-# Bitswap flows
+Bitswap flows
+=============
 
 Inside bitswap, there are two main flows: requesting blocks and serving blocks to other peers.
 
-## Requesting blocks
+Requesting blocks
+-----------------
 
-Client requests for new blocks are handled by the want manager, for every new block (or set of blocks) wanted, the 'WantBlocks' method is invoked. The want manager then ensures that connected peers are notified of the new block that we want by sending the new entries to a message queue for each peer. The message queue will loop while there is work available and do the following: 
+Client requests for new blocks are handled by the want manager, for every new block (or set of blocks) wanted, the `WantBlocks` method is invoked. The want manager then ensures that connected peers are notified of the new block that we want by sending the new entries to a message queue for each peer. The message queue will loop while there is work available and do the following:
 
-- a) Ensure it has a connection to its peer
-- b) Grab the message to be sent
-- c) Send it. 
+1.  Ensure it has a connection to its peer
+2.  Grab the message to be sent
+3.  Send it
```

Comment:
This will not auto enum in Github flavoured markdown

Reply:
Are you sure? Seems to indicate that it will on the spec [here](https://github.github.com/gfm/#example-254), at least if I'm understanding it correctly.

Reply:
You are right! :)